fractal_server/app/routes/api/v2/images.py

```diff
@@ -60,7 +60,7 @@ async def post_new_image(
         raise HTTPException(
             status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
             detail=(
-                f"Image.zarr_url '{new_image.zarr_url}' is not relative to"
+                f"Image.zarr_url '{new_image.zarr_url}' is not relative to "
```

Comment:
f"Cannot create image with `zarr_url` which is not relative to {dataset.zarr_dir}"